libsplinter/src/rest_api/auth/authorization/allow_keys.rs

```diff
@@ -406,4 +395,42 @@ mod tests {
             writeln!(file, "{}", key).expect("Failed to write key to file");
         }
     }
+
+    /// Verifies that the `AllowKeysAuthorizationHandler` is able to start without an existing file
+    /// and load the file once it's created.
+    ///
+    /// 1. Create a new temp directory
+    /// 2. Create a new `AllowKeysAuthorizationHandler` backed by a non-existent file in the temp
+    ///    directory
+    /// 3. Verify that `has_permission` returns `Continue`
+    /// 3. Create the backing file with a key
+    /// 4. Call `has_permission` with the key in the file and verify that `Allow` is returned
+    #[test]
+    fn load_after_file_created() {
+        let temp_dir = TempDir::new("file_removed").expect("Failed to create temp dir");
```

Comment:
Should this string be "load_after_file_created" instead of "file_removed"?

Reply:
Yep, fixed